dfget/core/api/supernode_api_test.go

```diff
@@ -21,24 +21,25 @@ import (
 	"strings"
 
 	"github.com/dragonflyoss/Dragonfly/common/constants"
+	cutil "github.com/dragonflyoss/Dragonfly/common/util"
 	"github.com/dragonflyoss/Dragonfly/dfget/types"
 
 	"github.com/go-check/check"
 )
 
```

Comment:
Add these codes into package `api` to run unit-test:
```go
func Test(t *testing.T) {
	check.TestingT(t)
}
```

Reply:
Done.